tidb-cloud: add incompatiblities between br and lightning (#15395)
pingcap/docs: tidb-cloud/backup-and-restore.md

## Limitations

- Although TiDB Dedicated supports restoring user accounts from backups by default, it does not support restoring system variables stored in the `mysql` schema. 

> - Recommend importing data first, then performing a **manual** snapshot backup, and finally enabling point-in-time restore. As the data imported through the TiDB Dedicated console **doesn't** generate change logs, it cannot be automatically detected and backed up. For more information, refer to [Import CSV Files from Amazon S3 or GCS into TiDB Cloud](/tidb-cloud/import-csv-files.md).

Reviewer: ```suggestion
- It is recommended that you import data first, then perform a **manual** snapshot backup, and finally enable Point-in-time restore. Because the data imported through the TiDB Dedicated console **does not** generate change logs, it cannot be automatically detected and backed up. For more information, see [Import CSV Files from Amazon S3 or GCS into TiDB Cloud](/tidb-cloud/import-csv-files.md).
```

> - It is recommended that you import data first, then perform a **manual** snapshot backup, and finally enable Point-in-time restore. Because the data imported through the TiDB Cloud console **does not** generate change logs, it cannot be automatically detected and backed up. For more information, see [Import CSV Files from Amazon S3 or GCS into TiDB Cloud](/tidb-cloud/import-csv-files.md).

Reviewer 2: ```suggestion
- It is recommended that you import data first, then perform a **manual** snapshot backup, and finally enable Point-in-time Restore. Because the data imported through the TiDB Cloud console **does not** generate change logs, it cannot be automatically detected and backed up. For more information, see [Import CSV Files from Amazon S3 or GCS into TiDB Cloud](/tidb-cloud/import-csv-files.md).
```
